editor/components/block-list/breadcrumb.js

```diff
@@ -62,14 +61,9 @@ export class BlockBreadcrumb extends Component {
 			} ) }>
 				<Toolbar>
 					{ rootUID && (
-						<IconButton
-							onClick={ selectRootBlock }
-							onFocus={ this.onFocus }
-							onBlur={ this.onBlur }
-							label={ __( 'Select parent block' ) }
-							icon="arrow-left-alt"
-						/>
+						<BlockTitle uid={ rootUID } />
 					) }
+					{ rootUID && ( <span> &rarr; </span> ) }
```

Comment:
Directionality has me wondering how this appears on RTL.

Reply:
Is the `span` necessary? Could this be `{ rootUID && ' → ' }` ?

_Edit:_ I see we're applying styles. We should probably do so with a class.

Reply:
>Directionality has me wondering how this appears on RTL.
>
>[...]
>
>Is the `span` necessary? Could this be `{ rootUID && ' → ' }` ?

Updated to an element with `:after` pseudo-class `content` (and RTL flipping) in e650954c389401426954b87972ea86d1857ff315.